src/main/java/io/excaliburfrc/robot/subsystems/Climber.java

```diff
@@ -39,32 +84,59 @@ public void close() {
     }
   }
 
-  public void activateLeader(MotorMode m) {
-    motor.set(m.dutyCycle);
+  public void activateMotors(MotorMode m) {
+    leftMotor.set(m.dutyCycle);
+    rightMotor.set(m.dutyCycle);
   }
 
   public void openAngler() {
-    anglerPiston.set(DoubleSolenoid.Value.kForward);
+    piston.set(DoubleSolenoid.Value.kForward);
   }
 
   public void closeAngler() {
-    anglerPiston.set(DoubleSolenoid.Value.kReverse);
+    piston.set(DoubleSolenoid.Value.kReverse);
+  }
+
+  private Command reachSideHeight(
+      SparkMaxPIDController controller, RelativeEncoder encoder, double height) {
+    return new TrapezoidProfileCommand(
+            new TrapezoidProfile(
+                new TrapezoidProfile.Constraints(
+                    feedforward.maxAchievableVelocity(12, kA),
+                    feedforward.maxAchievableAcceleration(12, kV)),
```

Comment:
shouldn't this be the other way around? kA for acceleration and kV for velocity?

Reply:
It needs the acceleration to calculate the velocity and vice versa
![image](https://user-images.githubusercontent.com/74930666/152122125-1e23a2b5-26ca-45d8-997a-a1bb3aae5d6e.png)